css/about.css

```diff
@@ -0,0 +1,199 @@
+.info {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 2rem;
+  background-image:
+    linear-gradient(to right, rgba(255, 255, 255, 0.904), rgba(255, 255, 255, 0.938)),
+    url(../img/Backgrounds/bg1.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.info h2 {
+  text-align: center;
+  color: #ec5442;
+  font-size: x-large;
+  font-weight: 300;
+  margin-bottom: 0.5rem;
+}
+
+.info h1 {
+  text-align: center;
+  font-family: "COCOGOOSE", sans-serif;
+  color: #ec5442;
+  font-size: xx-large;
+  margin-bottom: 0.5rem;
+}
+
+.info-descr {
+  text-align: center;
+  border: 3px solid #d3d3d3;
+  background-color: white;
+  padding: 1.5rem;
+  margin-bottom: 0.5rem;
+}
+
+.info-contact h3 {
+  text-align: center;
+  font-weight: 300;
+  font-size: 13px;
+  margin-bottom: 0.5rem;
+}
+
+.info-contact h4 {
+  text-align: center;
+  font-weight: 600;
+  font-size: 14px;
+  text-decoration: underline;
+}
+
+.the-logo {
+  text-align: center;
+  padding: 1.5rem;
+}
+
+.section-title {
+  margin-bottom: 1rem;
+}
+
+.the-logo p {
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+}
+
+.logo-conf {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #272a31;
+  padding: 2rem;
+  margin-bottom: 2em;
+}
+
+.logo-conf img {
+  width: 100%;
+  height: auto;
+}
+
+.past {
+  border-top: 1px solid rgb(201, 201, 201);
+  padding: 2rem;
+}
+
+.past p {
+  text-align: center;
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+}
+
+.year-box {
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 13rem;
+  width: 100%;
+}
+
+.year {
+  color: white;
+  font-weight: 900;
+  font-size: xx-large;
+}
+
+.year-box p {
+  color: white;
+}
+
+.bott {
+  margin-bottom: 2rem;
+}
+
+.year-fst {
+  background-image:
+    linear-gradient(to right, rgba(242, 109, 109, 0.6), rgba(242, 109, 109, 0.6)),
+    url(../img/album/luis-quintero.jpg);
+  background-size: cover;
+  background-position: center;
+}
+
+.year-sed {
+  background-image:
+    linear-gradient(to right, rgba(242, 109, 109, 0.6), rgba(242, 109, 109, 0.6)),
+    url(../img/album/matheus-bertelli.jpg);
+  background-size: cover;
+  background-position: center;
+}
+
+.year-thd {
+  background-image:
+    linear-gradient(to right, rgba(242, 109, 109, 0.6), rgba(242, 109, 109, 0.6)),
+    url(../img/album/mentatdgt.jpg);
+  background-size: cover;
+  background-position: center;
+}
+
+@media screen and (min-width: 700px) {
+  .info {
+    padding-left: 8rem;
+    padding-right: 8rem;
+    padding-top: 12rem;
+    padding-bottom: 5rem;
+  }
```

Comment:
It is best to properly organize all your media queries at the bottom of your stylesheet. So kindly move this to the bottom of your CSS

Reply:
Thank you for reviewing my project, and thank you for all your suggestions and encouragement. 
about this required change, I didn't get it well,  all the media queries are at the bottom of my css.